theory.tex

```diff
@@ -38,74 +38,114 @@ \subsubsection{Separative power}
+
+\begin{equation}
+    Z_p = \frac{(1-\theta)(1+L/F)}{1-\theta+L/F}Z
+\end{equation}
+
+This optimizes the recitifer length based on the cut, $\theta$, which is an expression
+of the fraction of the centrifuge feed that is output as product, and the counter-current
+flow, $L/F$. In practice, this value is a design parameter that is defined in the model
+during the design of the centrifuge cascade.
+
+The parameters $r_1$ and $r_2$ are the separation radii of the enriched material
+(here ${}^{235}\mathrm{U}$ vs. ${}^{238}\mathrm{U}$), $r_1$ being the withdrawal
+radius for the lighter isotope and $r_2$ for the heavier isotope. R\"aetz's
+two-shell model looks for optimal values between these two, as described by
+the hydrodynamic equations. The radii ratio is optimized using the following relationship:
+
+\begin{equation}
+    \max \left\{ \left[1-\left(\frac{r_1}{r_2}\right)^2 \right]^2 \times \left[ \ln \left(\frac{r_2}{r_1} \right) \right]^{-1} \right\}
+\end{equation}
+
+This ratio can further be constrained by approximating $r_2$ as being very close
+to the inner centrifuge wall with radius $a$:
+
+\begin{equation}
+    \left(\frac{r_1}{r_2}\right) \approx \left(\frac{r_1}{a}\right) = \sqrt{1 - \frac{2RT}{M}(\ln x)\frac{1}{v_{a}^{2}}}
+\end{equation}
+ 
+Here the gas constants are molar mass, $M$, temperature $T$, universal gas constant, $R$,
+and the pressure ratio, $x$ (typically $1000:1$) \cite{}. %needs citation?
+This relationship is valid when $v_a > 380 \frac{\mathrm{m}}{\mathrm{s}}$.
+Otherwise, the relationship can be approximated $\{\frac{r_1}{r_2} \approx 0.534 \mid v_a \leq 380 \frac{\mathrm{m}}{\mathrm{s}}\}$.
+In order to decompose the ratio into each individual radius, knowledge on one
+of them is required. Glaser \cite{glaser.2008} states that $r_2$ typically ranges
+from $96\%$ to $99\%$ the value of the inner centrifuge wall radius $a$. The exact
+behavior of $r_2$ between these two values in operational conditions is unknown,
+but a linear approximation can be made. It is assumed that $r_2$ is always at
+the midpoint of these two extremes (i.e. $r_2 = 0.975 a$). Then, $r_1$ can be
+found by simply multiplying this value by the ratio.
+
+With this, all parameters of the separative power equation \ref{eq:raetz} are
+defined and a value can be calculated. Separative power, $\delta U$, has units similar to the feed flow, that of $[mg/s]$.
+
 \paragraph{First principle}
 
-It can be shown \ref{avery} that the separative power of a single centrifuge can be
-derive from the first principle and expressed as a function of the feed to
-product enrichment factor, $\alpha$, the cut, $\theta$, and the feed rate, $F$:
+It can be shown \cite{avery} that the separative power of a single centrifuge can be
+derived from first principles and expressed as a function of the feed-to-product enrichment factor, $\alpha$, the cut, $\theta$, and the feed rate, $F$:
+%I think we should replicate at least some of that here
```

Comment:
Good point, although IIRC Avery only states this equation on its own. Will double check in the morning.

Reply:
Avery/Davies derive this relationship from two equations (page 18): (1) the definition of separative power, which is a function of delta_S (teh entropy change) and (2) the entropy. There's some variable changes to express entropy in terms of the cut (theta) and the enrichment factor (alpha) that leads to this equation. I'll try to express these equations in the paper this week (since I have the book).

Reply:
@nuclearkatie @bam241 I have expanded on the first principles derivation by repeating some of the equations in [Katie's worksheet](https://docs.google.com/document/d/1VoyoLFQV4TStAJnC33IAppMqRXd4VYxUScxVcxpuRGQ/edit?usp=sharing). I should stress that this is _far from a complete derivation_ and skips steps heavily. I think this is a good first step of addressing this but we can discuss further next time we meet.